Makefile

```diff
@@ -8,53 +8,83 @@ else
 	GULP_ENVIRONMENT := production
 endif
 
-run_all: requirements npm_install frontend_build run_app
+.PHONY: run-all
+run-all: requirements npm-install frontend-build run-app
 
-run_app: show_environment virtualenv
+.PHONY: run-app
+run-app: show-environment virtualenv
 	${VIRTUALENV_ROOT}/bin/python application.py runserver
 
+.PHONY: virtualenv
 virtualenv:
 	[ -z $$VIRTUAL_ENV ] && [ ! -d venv ] && virtualenv venv || true
 
-upgrade_pip: virtualenv
+.PHONY: upgrade-pip
+upgrade-pip: virtualenv
 	${VIRTUALENV_ROOT}/bin/pip install --upgrade pip
 
-requirements: virtualenv upgrade_pip requirements.txt
+.PHONY: requirements
+requirements: virtualenv test-requirements upgrade-pip requirements.txt
 	${VIRTUALENV_ROOT}/bin/pip install -r requirements.txt
 
-requirements_for_test: virtualenv upgrade_pip requirements_for_test.txt
-	${VIRTUALENV_ROOT}/bin/pip install -r requirements_for_test.txt
-
-npm_install: package.json
+.PHONY: requirements-dev
+requirements-dev: virtualenv test-requirements requirements-dev.txt
```

Comment:
That's what it used to be (`requirements_for_test.txt`). The reason for the rename was to reflect the fact that this should also be used for non-test related dev dependencies - like script dependencies or other tools we use that shouldn't be installed in live environments.